app/react/settings/people/PersonCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link, BrowserRouter } from "react-router-dom";
+
+const PersonCard = data => (
+    <BrowserRouter>
```

Comment:
Not sure we need BrowserRouter at this level. Would need to experiment around a bit though.

Reply:
You can't really do it without the browser router AFAIK, the link goes down in flames. Maybe I'm doing something wrong using link tho, need to recheck docs/egghead

Reply:
ah we didn't have one at all. Moved to peoplelistcontainer, but we should probably have a <main> somewhere for that sort of thing